contracts/src/systems/npc/utils.cairo

```diff
@@ -29,3 +31,21 @@ fn pedersen_hash_many(data: Span<felt252>) -> felt252 {
     };
     pedersen(current_hash, data.len().into())
 }
+
+fn format_data_to_hash(nonce: felt252, characteristics: felt252, character_trait: felt252, full_name: felt252) ->Span<felt252> {
```

Comment:
Could be a bit more explicit with the naming in this function for the function name and the data_array

Reply:
Done.